src/main/java/frc/team2412/robot/Subsystems.java

```diff
@@ -59,7 +59,9 @@ public Subsystems() {
         if (SHOOTER_ENABLED) {
             shooterSubsystem = new ShooterSubsystem();
             shooterVisionSubsystem = new ShooterVisionSubsystem();
-            targetLocalizer = new TargetLocalizer(drivebaseSubsystem, shooterSubsystem, shooterVisionSubsystem);
+            if (DRIVE_ENABLED) {
```

Comment:
As the documentation says, `TargetLocalizer` has null checks for drivebase (except for `getFieldCentricRobotAngle()`, which isn't called directly or indirectly and is fixed in the second pose estimation PR anyways).
In fact, these null checks were added so that we could remove this check and commands that require `TargetLocalizer` but not `DrivebaseSubsystem` could work if drivebase is disabled.